product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -53,6 +53,7 @@ The following are the default permissions by role:
 |               | events            | read                             |
 |               | identity provider | read                             |
 |               | permissions       | read                             |
+|               | regions           | activate, read, update, and deactivate | 
```

Comment:
@nstott  

Do the terms  'activate' and 'deactivate' work here ? Or should this doc stick with 'create' and 'delete'?